racket/collects/pkg/private/new.rkt

```diff
@@ -62,20 +62,229 @@
     (make-directory name)
     (parameterize ([current-directory name])
 
-      ;; LICENSE.txt
-      (with-output-to-file "LICENSE.txt"
+      ;; LICENSE files
+      (displayln "Generating LICENSE-APACHE and LICENSE-MIT files. You are free to change the license.")
+      (with-output-to-file "LICENSE-MIT"
+        (lambda () (expand/display #<<EOS
+MIT License
+
+Copyright (c) <<year>> <<name>> <<user>>
```

Comment:
I'm not sure what's the best way to deal with this line, but the lack of punctuation between `<<name>>` and `<<user>>` looks... incorrect.

```
Copyright (c) 2019 hello-world sorawee
```

Reply:
fixed - now has name on first line